treeherder/webapp/api/job_log_url.py

```diff
@@ -45,23 +38,3 @@ def list(self, request, project, jm):
         job_log_url_list = jm.get_job_log_url_list([job_id])
 
         return Response(job_log_url_list)
-
-    @detail_route(methods=['post'])
-    @with_jobs
-    def parse(self, request, project, jm, pk=None):
-        """Trigger an async task to parse this log."""
-
-        log_obj = jm.get_job_log_url_detail(pk)
-        job = jm.get_job(log_obj["job_id"])[0]
-
-        log_obj["job_guid"] = job["job_guid"]
-        # We can fake the result because there is only one queue for high priority
-        # jobs and it doesn't depend on the result
-        log_obj["result"] = None
-
-        logger.info("{0} has requested to parse log {1}".format(
-                    request.user.username, log_obj))
-
-        jm.schedule_log_parsing([log_obj], priority="high")
```

Comment:
Yeah, let's leave that for later... I'm actually not even inclined to necessarily file a bug, since I'm not sure when we'd ever get to it (at some point aren't we just going to stop log parsing?), but I can if you *really* want me to.